fe/fe-core/src/main/java/com/starrocks/mysql/privilege/AuthUpgrader.java

```diff
@@ -166,6 +167,7 @@ protected void upgradeUser() throws AuthUpgradeUnrecoverableException {
                     LOG.info("ignore entry created by domain resolver : {}", entry);
                     continue;
                 }
+                // collection of userIdentity
```

Comment:
This comment is misleading, `UserPrivilegeCollection` is not a collection of userIdentity, it represents the privilege collection a specified user has, including owned roles.